docs/manual/developer/03_creating_content.md

```diff
@@ -355,3 +355,372 @@ A profile should define these attributes:
+      unauthorized access to the system.
+    controls:
+      - id: R4.a
+        title: Disable administrator accounts
+        automated: yes
+        level: low
+        rules:
+          -  accounts_passwords_pam_faillock_deny_root
+      - id: R4.b
+        title: Enforce password quality standards
+        automated: yes
+        level: high
+        rules:
+          - accounts_password_pam_minlen
+          - accounts_password_pam_ocredit
+          - var_password_pam_ocredit=1
+```
+
+### Using controls in profiles
+
+Later, we can use the policy requirements in profile YAML. Let’s say that we
+will define a “Desktop” profile built from the controls.
+
+To use controls, we add them under `selection` keys in the profile. The entry
+has the form `policy_id:control_id`.
+
+```
+$ cat rhel8/profiles/abcd-desktop.profile
+ 
+documentation_complete: true
+title: ABCD Desktop for Red Hat Enterprise Linux 8
+description: |-
+  This profile contains configuration checks that align to
+  the ABCD benchmark.
+selections:
+  - abcd:R1
+  - abcd:R2
+  - abcd:R3
+  - security_patches_uptodate
+```
+
+Notice we can mix the controls selections with normal rule selections.
+
+In a similar way, we could define another profile that selects only some of the
+requirements.
+
+In the example we have selected all controls from `controls/abcd.yml` by listing
+them explicitly. It is possible to shorten it using the `“all”` value which
+means that all controls will be selected. Let’s show how it will be easier:
+
+```
+$ cat rhel8/profiles/abcd-high.profile
+ 
+documentation_complete: true
+title: ABCD High for Red Hat Enterprise Linux 8
+description: |-
+  This profile contains configuration checks that align to
+  the ABCD benchmark.
+selections:
+  - abcd:all
+  - security_patches_uptodate
+```
+
+It is possible to use levels if the levels are defined in the controls file. For
+example, `abcd:all:low` selects all rules for the ABCD low level or
+`abcd:all:high` selects all rules from the ABCD high level.
+
+Finally, when we build the content we will automatically get a SCAP profile
+which contains XCCDF rules and variables from all controls selected in profile
+YAML.
+
+The build system adds all XCCDF rules listed under `rules` key in the control to
+the built profile. The rules listed under `related_rules` key are not added.
+Therefore, the `related_rules` don't affect the generated source data stream.
+Also, the selections from `selection` key in profile file are included.
+
+In our example, the generated profile will contain rules
+`sshd_set_idle_timeout`, `accounts_tmout`, `var_accounts_tmout=10_min` and
+`security_patches_uptodate`. The profile will not contain
+`systemd_target_multi_user` even if control `R2` is selected because that is
+listed under `related_rules`.  The profile will be compiled to a canonical
+form.
+
+Example of a compiled profile:
+
+```
+$ cat build/rhel8/profiles/abcd-desktop.profile
```

Comment:
You might also want to call this out explicitly somewhere, but for debugging, knowing and listing this path explicitly (and how to find it) would be helpful, IMO.